importlab/resolve.py

```diff
@@ -151,11 +184,12 @@ def resolve_import(self, item):
         for fs in self.fs_path:
             for module_name, path in files:
                 f = self._find_file(fs, path)
-                if f:
-                    if item.is_relative():
-                        return Relative(f, module_name)
-                    else:
-                        return Local(f, module_name, fs)
+                if not f:
+                    continue
+                if item.is_relative():
+                    module_name = get_absolute_name(
+                            self.current_module.package_name, module_name)
+                return Local(f, module_name, fs)
```

Comment:
It seems a little odd to call this "Local" in the cases when it's relative to a System module.

Reply:
good point. done.